### PR TITLE
Update the Laravel docs for v2

### PR DESCRIPTION
**Pull request by mnapoli**
We are looking at making https://github.com/cachewerk/bref-laravel-bridge the default Laravel bridge instead of https://github.com/brefphp/laravel-bridge.

The reason for this is that the CacheWerk bridge implements more automatic configuration, supports Octane, supports all Laravel Queues features, etc.

This PR updates the Laravel docs based on this new bridge. It's in draft until we move the GitHub repository.

### CODE REVIEW

**Comment by mnapoli** [comment]
Related PR: https://github.com/brefphp/laravel-bridge/pull/94

File: docs/frameworks/laravel.md [hunk 1]
```diff
@@ -67,46 +67,34 @@ Follow [the deployment guide](/docs/deploy.md#deploying-for-production) for more
...
-Because Laravel is executed through API Gateway, the `Host` header is set to the API Gateway host name. Helper functions such as `redirect()` will use this incorrect domain name. The correct domain name is set on the `X-Forwarded-Host` header.
-
-To get Laravel to use `X-Forwarded-Host` instead, edit the `App\Http\Middleware\TrustProxies` middleware and set `$proxies` to the `*` wildcard:
-
-```php
-class TrustProxies extends Middleware
-{
-    // ...
-    protected $proxies = '*';
-```
```
Comment (mnapoli) [inline docs/frameworks/laravel.md:81]: @tillkruss I removed this section from the docs as the new bridge takes care of this, right?

Comment (tillkruss) [reply docs/frameworks/laravel.md:81]: I think so. I'll test this all once Bref v2 is out and I run into issues in production 🫠

Comment (tillkruss) [reply docs/frameworks/laravel.md:81]: Looks like it: https://github.com/cachewerk/bref-laravel-bridge/blob/0193eddb81819764553ed115b69e3ec65d43c058/src/BrefServiceProvider.php#L45

**Pull request merged**